cpp/include/cudf/lists/detail/stream_compaction.hpp

```diff
@@ -28,16 +28,18 @@ namespace cudf::lists::detail {
  * rmm::device_async_resource_ref)
  *
  * @param stream CUDA stream used for device memory operations and kernel launches
+ * @param mr Device memory resource used to allocate the returned column's device memory
```

Comment:
Leaving a link to the same comment from the other detail `stream_compaction.hpp` here to apply to this file as well.
https://github.com/rapidsai/cudf/pull/16295#discussion_r1684317000